app/routers/statistics.py

```diff
@@ -19,7 +20,7 @@ async def get_admin_statistics_by_lender(
     initial_date: str | None = None,
     final_date: str | None = None,
     lender_id: int | None = None,
-    custom_range: str | None = None,
+    custom_range: StatisticCustomRange | None = None,
```

Comment:
Now that we added this validation this is failing because the front end also uses CUSTOM_RANGE as one possible value

Reply:
From the back end logic, if custom_range = CUSTOM_RANGE then the initial date is none and the final date is set to today (so all time period)

Reply:
But it could also be CUSTOM_RANGE with an initial and final date provided, so I think we need to add CUSTOM_RANGE to StatisticCustomRange (and maybe rename it to StatisticRange)